pkg/controller/installation/products/amqstreams/reconciler.go

```diff
@@ -24,15 +24,15 @@ var (
 	defaultInstallationNamespace = "amq-streams"
 )
 
-func NewReconciler(client pkgclient.Client, rc *rest.Config, coreClient *kubernetes.Clientset, configManager config.ConfigReadWriter, instance *v1alpha1.Installation, mpm marketplace.MarketplaceInterface) (*Reconciler, error) {
+func NewReconciler(rc *rest.Config, coreClient *kubernetes.Clientset, configManager config.ConfigReadWriter, instance *v1alpha1.Installation, mpm marketplace.MarketplaceInterface) (*Reconciler, error) {
 	config, err := configManager.ReadAMQStreams()
 	if err != nil {
 		return nil, err
 	}
 	if config.GetNamespace() == "" {
 		config.SetNamespace(instance.Spec.NamespacePrefix + defaultInstallationNamespace)
 	}
-	return &Reconciler{client: client,
+	return &Reconciler{
 		coreClient:    coreClient,
 		restConfig:    rc,
```

Comment:
Can you check to see if the `*rest.Config` is still needed? I think that was related to the `serverClient`

Reply:
nice, yeah, removed that!